src/main.js

```diff
@@ -103,6 +105,53 @@ const load = loader({
     }),
   },
 
+  Queues: {
+    requires: ['cfg', 'process', 'monitor'],
+    setup: ({cfg, process, monitor}) => {
+      return data.Queues.setup({
+        tableName: cfg.app.queuesTableName,
+        monitor: monitor.prefix('table.queues'),
+        credentials: sasCredentials({
+          accountId: cfg.azure.accountId,
+          tableName: cfg.app.queuesTableName,
+          rootUrl: cfg.taskcluster.rootUrl,
+          credentials: cfg.taskcluster.credentials,
+        }),
+        //cryptoKey: cfg.azure.cryptoKey,
```

Comment:
@Biboswan found a similar thing -- this table doesn't need any encrypted columns, so no need for encryption.

Reply:
Yes, that's why I commented it :)
Will remove it in the next commit